nima_core/core.py

```
@@ -153,6 +153,42 @@ def _initialize(self):
             except Exception as e:
                 logger.warning(f"Metacognitive init failed: {e}")
+        
+        # DARING Engine (response modulation)
+        try:
+            from .cognition.daring_engine import DaringEngine
+            self._daring = DaringEngine(
+                affective_core=self._affective if hasattr(self, '_affective') else None,
+                data_dir=str(self.data_dir / 'daring')
+            )
+            logger.info("DARING Engine initialized")
+        except Exception as e:
```

Ruff (0.15.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)